src/homepage/_posts/2021-03-16-james-3.6.0.markdown

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title:  "Apache James Server 3.6.0"
+date:   2021-03-16  15:16:30 +0200
+categories: james update
+---
+
+The Apache James developers are pleased to announce James server 3.6.0 release.
+
+Early adopters can [download it][download], any issue can be reported on our issue [tracker][tracker].
+
+## Announcements
+
+### Experimental support for JMAP RFC-8621 and deprecation of JMAP Draft
+
+Since the last release we worked hard refreshing our JMAP implementation, implementing both [JMAP core][jmap-core],
+[JMAP mail][jmap-mail] and the [JMAP over WebSocket][jmap-ws] specifications. Their support is considered experimental and
+early adopters' feedback is highly welcomed.
+
+[JMAP Draft][jmap-draft] should be considered legacy, and will be removed in next releases.
+
+## Release changelog
+
+Here are some points we worked on:
+
+### Added
+- JAMES-2884 Partial Support for JMAP RFC-8621: The current implementation status allow reading mailboxes, emails, vacation responses.
+  - JAMES-3457 Implement JMAP eventSource 
+  - JAMES-3491 JMAP over websocket (RFC-8887)
+  - JAMES-3470 JMAP RFC-8621 Email/changes + Mailbox/changes support
+- JAMES-3117 Add PeriodicalHealthChecks for periodical calling all health checks
+- JAMES-3143 WebAdmin endpoint to solve Cassandra message inconsistencies
+- JAMES-3138 Webadmin endpoint to recompute users current quotas on top of Guice products
+- JAMES-3296 Webadmin endpoint to rebuild RabbitMQMailQueue in the Distributed Server
+- JAMES-3266 Offer an option to disable ElasticSearch in Distributed James product
+- JAMES-3202 Reindex only outdated documents with the Mode option set to CORRECT in reindexing tasks
+- JAMES-3405 Expose metrics of Guice servers over HTTP - enables easy Prometheus metrics collection
+- JAMES-3407 Distributed server: Read-repairs for the mailbox entity
+- JAMES-3428 Distributed server: Read-repairs for the mailbox counters entity
+- JAMES-3139 Expose RabbitMQ channel & connection configuration
+- JAMES-3441 Make possible and document Distributed Server setup with specialized instances
+- JAMES-3337 Document the use of JWT
+- JAMES-3399 Allow JSON logging with logback - enables structure logging with FluentBit
+- JAMES-3396 WebAdmin should try to prevent RRT addresses redirection loops when possible
+- JAMES-3402 JMAP MDN messages should have a Date header
+- JAMES-3028 Distributed server: allow choosing whether blobs should be deduplicated
+- JAMES-3196 CanSendFromImpl: enable to send email from aliases for SMTP and JMAP
+- JAMES-3196 Add an IMAP SessionId to correlate logs
+- JAMES-3502 DistributedServer: SSL and authentication support for RabbitMQ
+- JAMES-3504 Metrics and log for POP3
+- JAMES-3431 Optional DSN support
+- JAMES-3202 Allow search index Reindexing without cleanup
+
+### Changed
+- Switch to Java 11 for build and run
```

Comment:
Isn't this big enough to warrant an entry in the Announcements ? (it could have large impacts for deployed systems)